autopilot/ap-batch-transform.ipynb

```diff
@@ -0,0 +1,1297 @@
+{
```

Comment:
<p>Need to add badges. Just follow the instructions under "Add CI badges to your notebook" in&nbsp;<a href="https://github.com/aws/amazon-sagemaker-examples/blob/main/CONTRIBUTING.md" target="_blank">https://github.com/aws/amazon-sagemaker-examples/blob/main/CONTRIBUTING.md</a></p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/4312/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='0'/>

Reply:
added